src/sseq.rs

```diff
@@ -96,9 +96,56 @@ impl<const N: usize> SSeqGen<N> {
         res
     }
 
+    /// Iterator over sequences one substitution (hamming distance) away.
     pub fn one_hamming_iter(self, opt: HammingIterOpt) -> SSeqOneHammingIter<N> {
         SSeqOneHammingIter::new(self, opt)
     }
+
+    /// Iterator over sequences one deletion away.
+    pub fn one_deletion_iter(self) -> impl Iterator<Item = Self> {
+        (0..self.len()).map(move |i| {
+            let mut seq = self;
+            seq.remove(i);
+            seq
+        })
+    }
+
+    /// Iterator over sequences one insertion away. The length of the sequence
+    /// should be less than the capacity so that there is room for 1 insertion
```

Comment:
```suggestion
    /// Iterator over sequences one insertion away. The length of the sequence
    /// must be less than the capacity so that there is room for 1 insertion.
```